### PR TITLE
Update runtime version to 18.08

### PR DESCRIPTION
**Pull request by Erick555**

### CODE REVIEW

**Comment by wjt** [comment]
I'm afraid that something in my Vim configuration still causes this to segfault as soon as I enter insert mode. (If I move my `~/.vim` out of the way it works fine.)

```
#0  0x00007fffedc578ad in PyModule_Create2 () from /usr/lib/x86_64-linux-gnu/libpython3.7m.so.1.0
#1  0x00007ffff5c49be6 in PyInit__heapq () from /usr/lib/python3.7/lib-dynload/_heapq.cpython-37m-x86_64-linux-gnu.so
#2  0x00005555558a5294 in ?? ()
#3  0x00005555558a330f in ?? ()
#4  0x000055555580086f in ?? ()
#5  0x00005555558009f9 in ?? ()
#6  0x00005555555d89a2 in ?? ()
#7  0x000055555588cd7e in ?? ()
#8  0x00005555557ffcd7 in ?? ()
#9  0x00005555555d7c3d in ?? ()
#10 0x00005555555cf47b in ?? ()
#11 0x00005555555d803d in ?? ()
#12 0x00005555555cf47b in ?? ()
#13 0x00005555555d6b77 in ?? ()
#14 0x00005555555cf47b in ?? ()
#15 0x00005555555d6b77 in ?? ()
#16 0x00005555555cf47b in ?? ()
#17 0x00005555555d6b77 in ?? ()
#18 0x00005555555cf47b in ?? ()
#19 0x00005555557ffc2b in ?? ()
#20 0x00005555558018c0 in ?? ()
#21 0x0000555555801b51 in ?? ()
#22 0x00005555558a4397 in ?? ()
#23 0x00005555555d731a in ?? ()
#24 0x000055555588cd7e in ?? ()
#25 0x000055555588cff2 in ?? ()
#26 0x000055555588d01f in ?? ()
#27 0x000055555588a095 in ?? ()
#28 0x000055555580086f in ?? ()
#29 0x00005555558009f9 in ?? ()
#30 0x00005555555d89a2 in ?? ()
#31 0x000055555588cd7e in ?? ()
#32 0x00005555557ffcd7 in ?? ()
#33 0x00005555555d7c3d in ?? ()
#34 0x00005555555cf47b in ?? ()
#35 0x00005555555d803d in ?? ()
#36 0x00005555555cf47b in ?? ()
#37 0x00005555555d6b77 in ?? ()
#38 0x00005555555cf47b in ?? ()
#39 0x00005555555d6b77 in ?? ()
--Type <RET> for more, q to quit, c to continue without paging--
#40 0x00005555555cf47b in ?? ()
#41 0x00005555557ffc2b in ?? ()
#42 0x00005555558018c0 in ?? ()
#43 0x0000555555801b51 in ?? ()
#44 0x00005555558a4397 in ?? ()
#45 0x00005555555d731a in ?? ()
#46 0x000055555588cd7e in ?? ()
#47 0x000055555588cff2 in ?? ()
#48 0x000055555588d01f in ?? ()
#49 0x000055555588a095 in ?? ()
#50 0x000055555580086f in ?? ()
#51 0x00005555558009f9 in ?? ()
#52 0x00005555555d89a2 in ?? ()
#53 0x000055555588cd7e in ?? ()
#54 0x00005555557ffcd7 in ?? ()
#55 0x00005555555d7c3d in ?? ()
#56 0x00005555555cf47b in ?? ()
#57 0x00005555555d803d in ?? ()
#58 0x00005555555cf47b in ?? ()
#59 0x00005555555d6b77 in ?? ()
#60 0x00005555555cf47b in ?? ()
#61 0x00005555555d6b77 in ?? ()
#62 0x00005555555cf47b in ?? ()
#63 0x00005555557ffc2b in ?? ()
#64 0x00005555558018c0 in ?? ()
#65 0x0000555555801b51 in ?? ()
#66 0x00005555558a4397 in ?? ()
#67 0x00005555555d731a in ?? ()
#68 0x000055555588cd7e in ?? ()
#69 0x000055555588cff2 in ?? ()
#70 0x000055555588d01f in ?? ()
#71 0x000055555588a095 in ?? ()
#72 0x000055555580086f in ?? ()
#73 0x00005555558009f9 in ?? ()
#74 0x00005555555d89a2 in ?? ()
#75 0x000055555588cd7e in ?? ()
#76 0x00005555557ffcd7 in ?? ()
#77 0x00005555555d7c3d in ?? ()
#78 0x00005555555cf47b in ?? ()
#79 0x00005555555d803d in ?? ()
--Type <RET> for more, q to quit, c to continue without paging--
#80 0x00005555555cf47b in ?? ()
#81 0x00005555555d6b77 in ?? ()
#82 0x00005555555cf47b in ?? ()
#83 0x00005555555d6b77 in ?? ()
#84 0x00005555555cf47b in ?? ()
#85 0x00005555557ffc2b in ?? ()
#86 0x00005555558018c0 in ?? ()
#87 0x0000555555801b51 in ?? ()
#88 0x00005555558a4397 in ?? ()
#89 0x00005555555d731a in ?? ()
#90 0x000055555588cd7e in ?? ()
#91 0x000055555588cff2 in ?? ()
#92 0x000055555588d01f in ?? ()
#93 0x000055555588a095 in ?? ()
#94 0x000055555580086f in ?? ()
#95 0x00005555558009f9 in ?? ()
#96 0x00005555555d89a2 in ?? ()
#97 0x000055555588cd7e in ?? ()
#98 0x00005555557ffcd7 in ?? ()
#99 0x00005555555d7c3d in ?? ()
#100 0x00005555555cf47b in ?? ()
#101 0x00005555555d803d in ?? ()
#102 0x00005555555cf47b in ?? ()
#103 0x00005555555d6b77 in ?? ()
#104 0x00005555555cf47b in ?? ()
#105 0x00005555555d6b77 in ?? ()
#106 0x00005555555cf47b in ?? ()
#107 0x00005555557ffc2b in ?? ()
#108 0x00005555558018c0 in ?? ()
#109 0x0000555555801b51 in ?? ()
#110 0x00005555558a4397 in ?? ()
#111 0x00005555555d731a in ?? ()
#112 0x000055555588cd7e in ?? ()
#113 0x000055555588cff2 in ?? ()
#114 0x000055555588d01f in ?? ()
#115 0x000055555588a095 in ?? ()
#116 0x000055555580086f in ?? ()
#117 0x00005555558009f9 in ?? ()
#118 0x00005555555d89a2 in ?? ()
#119 0x000055555588cd7e in ?? ()
--Type <RET> for more, q to quit, c to continue without paging--
#120 0x00005555557ffcd7 in ?? ()
#121 0x00005555555d7c3d in ?? ()
#122 0x00005555555cf47b in ?? ()
#123 0x00005555555d803d in ?? ()
#124 0x00005555555cf47b in ?? ()
#125 0x00005555555d6b77 in ?? ()
#126 0x00005555555cf47b in ?? ()
#127 0x00005555555d6b77 in ?? ()
#128 0x00005555555cf47b in ?? ()
#129 0x00005555557ffc2b in ?? ()
#130 0x00005555558018c0 in ?? ()
#131 0x0000555555801b51 in ?? ()
#132 0x00005555558a4397 in ?? ()
#133 0x000055555588827c in ?? ()
#134 0x0000555555801d7b in ?? ()
#135 0x00005555555d89a2 in ?? ()
#136 0x000055555588cd7e in ?? ()
#137 0x00005555557ffcd7 in ?? ()
#138 0x00005555555d6b77 in ?? ()
#139 0x000055555588cd7e in ?? ()
#140 0x00005555557ff9f2 in ?? ()
#141 0x00005555558018c0 in ?? ()
#142 0x0000555555801b51 in ?? ()
#143 0x00005555558a4466 in ?? ()
#144 0x00005555555d731a in ?? ()
#145 0x000055555588cd7e in ?? ()
#146 0x000055555588cff2 in ?? ()
#147 0x000055555588d01f in ?? ()
#148 0x000055555588a095 in ?? ()
#149 0x000055555580086f in ?? ()
#150 0x00005555558009f9 in ?? ()
#151 0x00005555555d89a2 in ?? ()
#152 0x000055555588cd7e in ?? ()
#153 0x00005555557ffcd7 in ?? ()
#154 0x00005555555d7c3d in ?? ()
#155 0x00005555555cf47b in ?? ()
#156 0x00005555555d803d in ?? ()
#157 0x00005555555cf47b in ?? ()
#158 0x00005555555d6b77 in ?? ()
#159 0x00005555555cf47b in ?? ()
--Type <RET> for more, q to quit, c to continue without paging--
#160 0x00005555555d6b77 in ?? ()
#161 0x00005555555cf47b in ?? ()
#162 0x00005555557ffc2b in ?? ()
#163 0x00005555558018c0 in ?? ()
#164 0x0000555555801b51 in ?? ()
#165 0x00005555558a4397 in ?? ()
#166 0x00005555555d731a in ?? ()
#167 0x00005555555cf47b in ?? ()
#168 0x00005555555d6b77 in ?? ()
#169 0x00005555555cf47b in ?? ()
#170 0x00005555555d6b77 in ?? ()
#171 0x000055555588cd7e in ?? ()
#172 0x000055555588cff2 in ?? ()
#173 0x000055555588d01f in ?? ()
#174 0x000055555588a095 in ?? ()
#175 0x000055555580086f in ?? ()
#176 0x00005555558009f9 in ?? ()
#177 0x00005555555d89a2 in ?? ()
#178 0x000055555588cd7e in ?? ()
#179 0x00005555557ffcd7 in ?? ()
#180 0x00005555555d7c3d in ?? ()
#181 0x00005555555cf47b in ?? ()
#182 0x00005555555d803d in ?? ()
#183 0x00005555555cf47b in ?? ()
#184 0x00005555555d6b77 in ?? ()
#185 0x00005555555cf47b in ?? ()
#186 0x00005555555d6b77 in ?? ()
#187 0x00005555555cf47b in ?? ()
#188 0x00005555557ffc2b in ?? ()
#189 0x00005555558018c0 in ?? ()
#190 0x0000555555801b51 in ?? ()
#191 0x00005555558a4397 in ?? ()
#192 0x000055555588827c in ?? ()
#193 0x00005555558007df in ?? ()
#194 0x00005555558009f9 in ?? ()
#195 0x0000555555800e46 in ?? ()
#196 0x0000555555800faa in ?? ()
#197 0x00005555558a4a1e in ?? ()
#198 0x00005555558a4c0e in ?? ()
#199 0x00005555558ac900 in ?? ()
--Type <RET> for more, q to quit, c to continue without paging--
#200 0x00005555558af168 in ?? ()
#201 0x00005555558af37b in ?? ()
#202 0x00005555558af415 in ?? ()
#203 0x00005555558af447 in ?? ()
#204 0x00005555557bdc75 in ?? ()
#205 0x00005555557c4ce5 in ?? ()
#206 0x0000555555644ae1 in ?? ()
#207 0x000055555564699d in ?? ()
#208 0x000055555560c56c in ?? ()
#209 0x0000555555644ae1 in ?? ()
#210 0x000055555564699d in ?? ()
#211 0x0000555555636791 in ?? ()
#212 0x0000555555635080 in ?? ()
#213 0x00005555556351f7 in ?? ()
#214 0x00005555556034a6 in ?? ()
#215 0x0000555555603622 in ?? ()
#216 0x00005555556038ed in ?? ()
#217 0x0000555555785c8e in ?? ()
#218 0x0000555555787e8c in ?? ()
#219 0x000055555578a104 in ?? ()
#220 0x0000555555644ae1 in ?? ()
#221 0x000055555564699d in ?? ()
#222 0x000055555565fe36 in ?? ()
#223 0x0000555555660c18 in ?? ()
#224 0x00005555555edf90 in ?? ()
#225 0x00005555555fa25d in ?? ()
#226 0x00005555556b92a6 in ?? ()
#227 0x00005555556b9e00 in ?? ()
#228 0x00005555556c5ef3 in ?? ()
#229 0x00005555557df25a in ?? ()
#230 0x00005555557e051f in ?? ()
#231 0x00005555555cd4c7 in ?? ()
#232 0x00007ffff6c9803b in __libc_start_main () from /usr/lib/x86_64-linux-gnu/libc.so.6
#233 0x00005555555d8a1a in ?? ()
```

It's caused by "ultisnips" and looks suspiciously similar to https://github.com/SirVer/ultisnips/issues/1032.

File: org.vim.Vim.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "app-id": "org.vim.Vim",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "1.6",
+  "runtime-version": "18.08",
   "sdk": "org.freedesktop.Sdk",
+  "branch": "stable",
```
Comment (wjt) [inline org.vim.Vim.json]: This is not needed (Flathub passes `--default-branch=stable`) and makes it harder to distinguish a locally-compiled development version from the Flathub release. I deliberately removed it in e672374242b510ef3000db86460c0c078fb8a8b8.

**Comment by Erick555** [comment]
It seems utilsnips upstream issue. I don't know what we can do about it. You mat try running it through Sdk to catch better backtrace.

File: org.vim.Vim.json (hunk 1 above)
Comment (Erick555) [reply org.vim.Vim.json]: Ok, feel free to remove this or I could do it later.

**Comment by Erick555** [comment]
Perfect! I assume we're good to go with this PR then?

**Pull request merged**